src/applications/covid19-chatbot/createCoronavirusChatbot.js

```diff
@@ -0,0 +1,15 @@
+export default (_store, widgetType) => {
```

Comment:
we should move this to coronavirus-chatbot directory for consistency

Reply:
Oh good catch, I think `src/applications/static-pages/coronavirus-chatbot-utils.js` should be moved into this directory to house the full widget code

Reply:
Will do!

Reply:
Removed for now-- since it's not used yet (contains logic for disabling buttons)